contracts/zero-ex/contracts/src/features/TransformERC20Feature.sol

```diff
@@ -313,7 +313,7 @@ contract TransformERC20Feature is
             to.transfer(msg.value);
         }
         // Transfer input tokens.
-        if (!LibERC20Transformer.isTokenETH(inputToken)) {
+        if (!LibERC20Transformer.isTokenETH(inputToken) && amount != 0) {
```

Comment:
This allows for a "transfer then execute" pattern for `_transformERC20`

Reply:
not urgent to redeploy this feature either; I doubt 0x API will be making use of this any time soon